webrtc.html

```diff
@@ -214,8 +214,25 @@
               </tr>
               <tr>
                 <td><dfn><code>token</code></dfn></td>
-                <td>The credential is an access token, as described in
-                [[!TRAM-TURN-THIRD-PARTY-AUTHZ]], Section 6.2.</td>
+                <td>
+                  <p>The credential is an OAuth 2.0 Access Token.</p>
+                  <p>For authentication, at least these parameters are required
+                     , as described in [[!TRAM-TURN-THIRD-PARTY-AUTHZ]]</p>
+                  <ul>
+                    <li>
+                      The kid (the Access Token encrpytion key ID). Handover
```

Comment:
Nit: "Handover it as" is not correct English. "This is passed in the API field called" might be.